.vscode/launch.json

```diff
@@ -1,19 +1,15 @@
 {
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
     "configurations": [
         {
-            "name": "Docker: Python - General",
-            "type": "docker",
```

Comment:
I decided not to debug as Docker and instead as current file. The docker debugging added additional headache of docker build every time. So the below file `settings.json` and `tasks.json` is not needed anymore